### ISSUE TITLE
Release 1.9.0 version

### ISSUE THREAD

**Issue by imincik** [linz/linz-lds-bde-schema#207]
Last tag: debian/1.8.2-1linz_bionic1

New changes:
```
commit 5367dabdd826914ee2f160893a0f5bb5b3158ae6
Author: Sandro Santilli <strk@kbt.io>
Date:   Thu Feb 6 16:37:58 2020 +0100

    Do not TRUNCATE an empty table (what's the point?) (#206)
    
    As of table_version 1.8.0+ TRUNCATE will be forbidden on versioned
    tables, so better not do it when not needed...

commit 936d18b3aa6e39d807a23ead5316ac1461ef4428
Author: Ivan Mincik <imincik@linz.govt.nz>
Date:   Tue Feb 4 11:16:11 2020 +0100

    [debian] Changelog update

commit 1e99af4373cff4492d7bf473d6ffc6c365521793
Author: Ivan Mincik <imincik@linz.govt.nz>
Date:   Mon Nov 18 16:15:10 2019 +0100

    [debian] Changelog update

commit 4b231615c9e106936d3b5562cc2681929b1d79cf
Author: Sandro Santilli <strk@kbt.io>
Date:   Tue Feb 4 10:07:55 2020 +0100

    Add 1.8.2 section

commit c095476971a2ceaed599ff47326dffbe72424b41
Author: Sandro Santilli <strk@kbt.io>
Date:   Thu Jan 30 10:50:24 2020 +0100

    Include more WACA adjustments
    
    References #174 for the master branch

commit 7b1f49d2b6d2515c312c3c14adedd258f3a855c5
Author: Sandro Santilli <strk@kbt.io>
Date:   Wed Jan 29 11:15:13 2020 +0100

    Include 1.8.1 section in CHANGELOG

commit 8196ebc9e2c29b7ee5940921a0fef5c5fac37f6e
Author: Sandro Santilli <strk@kbt.io>
Date:   Wed Jan 29 10:08:56 2020 +0100

    Catch more WACA adjustments
    
    Closes #174

commit 947c47703f20243cd87fc8288321d89c0b13cc53
Author: Sandro Santilli <strk@kbt.io>
Date:   Wed Jan 22 10:36:55 2020 +0100

    Ensure sequences used in serial columns are owned by them
    
    Fixes #196 for master branch
    
    Also targets 1.9.0dev

commit 48460b74d8fbc4381bc513acb2edb4beb737fa9d
Author: Sandro Santilli <strk@kbt.io>
Date:   Wed Jan 22 11:17:37 2020 +0100

    Add tests for sequence ownership

commit a79c34a6e1eabe46f54bc388932e0c782374e01f
Author: Ivan Mincik <imincik@linz.govt.nz>
Date:   Mon Nov 18 16:23:18 2019 +0100

    [debian] Add python to build dependences

commit ba881df70bf63e962190ce25c1649835f6fbf81b
Author: Ivan Mincik <imincik@linz.govt.nz>
Date:   Mon Nov 18 16:09:34 2019 +0100

    [debian] Changelog update
```

**Comment by strk** [comment]
It's really just the TRUNCATE at this moment, because earlier commits already landed in 1.8.2 which was packaged and tested in production. Still worth a new release, you think ?

**Comment by strk** [comment]
Well, released: https://github.com/linz/linz-lds-bde-schema/releases/tag/1.9.0
